x-pack/test/security_solution_cypress/cypress/e2e/investigations/sourcerer/create_runtime_field.cy.ts

```diff
@@ -44,13 +43,12 @@ describe.skip('Create DataView runtime field', { tags: ['@ess', '@serverless'] }
     cy.get(GET_DATA_GRID_HEADER(alertRunTimeField)).should('exist');
   });
 
-  it('adds field to timeline', () => {
+  // TODO: need to update refetch properly in the timeline tab
```

Comment:
should we log a ticket for this?

Reply:
Waiting on a response from @lgestc here as he fixed this in a separate PR here: https://github.com/elastic/kibana/pull/196827#discussion_r1814106496

Reply:
Pulled in his changes